ConditionsService/inc/ConditionsService.hh

```diff
@@ -123,5 +123,5 @@ namespace mu2e {
 
 }
 
-DECLARE_ART_SERVICE(mu2e::ConditionsService, LEGACY)
+DECLARE_ART_SERVICE(mu2e::ConditionsService, SHARED)
```

Comment:
I am confused about this change in several ways.  

First, I was surprised to see it made SHARED. 

Second, I cloned both Mu2e::master and goodenou:art3_MT locally.  And in both of my clones it shows as LEGACY?

Does anyone understand what's up?